native/tests/parity/compare_pcm.py

```
@@ -0,0 +1,66 @@
+    raise ValueError("choose exact=True or min_snr=<dB>")
+
+
+def _read_wav(path: str) -> np.ndarray:
+    import soundfile as sf
+    data, _sr = sf.read(path, dtype="float32", always_2d=False)
+    return data if data.ndim == 1 else data.mean(axis=1)
+
+
+def main(argv: list[str] | None = None) -> int:
+    p = argparse.ArgumentParser(description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter)
+    p.add_argument("ref"); p.add_argument("got")
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)